api/nakadi-event-bus-api.yaml

```diff
@@ -613,6 +632,11 @@ paths:
             The flow id of the request, which is written into the logs and passed to called services. Helpful
             for operational troubleshooting and log analysis.
           type: string
+        - name: consumed_offset
```

Comment:
There might be systems reasons for not doing so, but would it be reasonable to just send back `unconsumed_events` in the data as an additive field? As a client dev `unconsumed_events` sounds like something I'd going to want to have as default information to help with instrumentation on the consumer side.